.github/workflows/integration.yml

```diff
@@ -68,6 +68,8 @@ jobs:
       run: ./integration_tests.py test-client || touch client-tests-failed
     - name: Elasticsearch logs
       run: docker logs elasticsearch
+    - name: IAM logs
+      run: docker logs iam-login-service
```

Comment:
For my info, why is this needed?

Reply:
Well, it is not essential actually.
Should I remove these lines?